tests/src/test/java/alluxio/client/AbstractFileOutStreamIntegrationTest.java

```diff
@@ -62,11 +63,12 @@
 
   @Before
   public void before() throws Exception {
-    mWriteBoth = StreamOptionUtils.getCreateFileOptionsCacheThrough();
-    mWriteAlluxio = StreamOptionUtils.getCreateFileOptionsMustCache();
-    mWriteUnderStore = StreamOptionUtils.getCreateFileOptionsThrough();
-    mWriteLocal = StreamOptionUtils.getCreateFileOptionsWriteLocal();
-    mWriteAsync = StreamOptionUtils.getCreateFileOptionsAsync();
+    mWriteBoth = CreateFileOptions.defaults().setWriteType(WriteType.CACHE_THROUGH);
```

Comment:
These CreateFileOptions fields are named inconsistently/confusingly, so it's probably better to inline these as well. They are mostly used in just a couple of places.